Python/AmitShrestha/show_result.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import datetime
+
+parser = argparse.ArgumentParser(description="Result Parser")
+parser.add_argument("--store", required=True, help="File Name")
+
+
+def main(args):
+    file_name = vars(args)["store"]
```

Comment:
try direct attribute access or `getattr` `hasattr` route rather than converting to dict in terms of efficiency.